config/betterquesting/DefaultQuests.json

```diff
@@ -335210,9 +335230,9 @@
       "home_image:8": "dreamcraft:textures/gui/gtnh_title.png",
       "party_enable:1": 1,
       "pack_version:3": 2110,
-      "editMode:1": 0,
+      "editMode:1": 1,
```

Comment:
please disable edit mode or set this back to 0

Reply:
Done, thanks.